install.sh

```diff
@@ -1,26 +1,63 @@
 #!/bin/sh
```

Comment:
Do you know these tools?

- [shellcheck](https://github.com/koalaman/shellcheck)
- [shfmt](https://github.com/mvdan/sh)

They can help a lot when writing shell script

Reply:
I do, I've been using shellcheck for a while now. Thanks for the suggestions.

Reply:
Great thing then

These are game changers

Do you use shfmt?

Reply:
I was looking into it... and mate what a discovery this is! I've been writing shell script for years as a hobby, but this is amazing! It also allows you to minify shell script code?! Language dialect selection and much more?! Mate, you opened a new world right now!

Reply:
You're welcome.

Dev is a drug, I'm happy to support you getting or better (it's never easy to decide)

My moto is somehow: 

> Give a Man a Fish, and You Feed Him for a Day. Teach a Man To Fish, and You Feed Him for a Lifetime